doc/pac/Port Access Control.md

```diff
@@ -0,0 +1,804 @@
+| User Name                                | The user name associated with the client. |
+| VLAN                                     | The VLAN associated with the client.     |
+| Host Mode                                | The authentication host mode configured on the interface. The possible values are multi-auth, multi-host and single-host. |
+| Method                                   | The method used to authenticate the client on the interface. The possible values are dot1x or MAB. |
+| Session Time                             | The amount of time the client session has been active. |
+| Session Timeout                          | This value indicates the time for which the given session is valid. The time period in seconds is returned by the RADIUS server on authentication of the port. |
+| Time left for Session Termination Action | This value indicates the time left for the session termination action to occur. This field is valid only when the “authentication periodic” is configured. |
+| Session Termination Action               | This value indicates the action to be taken once the session timeout expires. Possible values are Default and Radius-Request. If the value is Default, the session is terminated and client details are cleared. If the value is Radius-Request, then a reauthentication of the client is performed. |
+
+
+
+**show mab <cr | interface\>**
+
+This command is used to show a summary of the global mab configuration and summary information of the mab configuration for all ports. This command also provides the detailed mab configuration for a specified port
+
+| Field         | Description                              |
+| ------------- | ---------------------------------------- |
+| Interface     | Given interface                          |
+| Admin Mode    | MAB admin mode on the given interface    |
+| MAB auth type | MAB authentication type (EAP_MD5, PAP, CHAP) |
+
+
+
+**show dot1x**
+
+This command is used to show a summary of the global 802.1x configuration.
+
+| Field               | Description                              |
+| ------------------- | ---------------------------------------- |
+| Administrative Mode | Indicates whether 802.1x is enabled or disabled. |
+
+
+
+
+show dot1x detail <all | interface\>
+
+This command is used to show details of 802.1x configuration on an interface.
+
+| Field            | Description                              |
+| ---------------- | ---------------------------------------- |
+| Interface        | Given Interface                          |
+| PAE Capabilities | The Port Access entity (PAE) functionality of this port. Possible values are Authenticator or None |
+
+
+
+### 3.7.4 Clear Commands
+
+**sonic-clear authentication sessions <interface <all | interface | mac \>\>**
+
+This command clears information for all Auth Manager sessions. All the authenticated clients are re-initialized and forced to authenticate again.
+
+
+
+# 4 Scalability
+
```

Comment:
Unless this is defined by the SW data construct. Scalability performance being HW dependent, I would suggest to remove this claim from the HLD

Reply:
Yes, these are SW defined constraints.

Reply:
Can we make thus scaling numbers configurable ?

Reply:
We want keep an upper limit which we can qualify. Hence these hard non-configurable limits.

Reply:
@samitabh Hard non-configurable limits might not be future scalable. Consider adding switch limits to this item and you can still qualify the switch limit currently supported.

Reply:
We do have configurable max users on a port from 1 to a MAX value. Why would we want to have MAX itself configurable ?

Reply:
For future use case to support more than 128 clients per port, you need to modify the MAX value. This can be avoided if this is configurable.

Reply:
Sure. We will reevaluate this in a future release.